README.md

```diff
@@ -24,18 +24,21 @@ lookup (we use geolocation of a named location) please [open an issue](https://g
 ### 2. How do I add a job?
 
 We maintain a list of current and previous job postings in [_data/jobs.yml](_data/jobs.yml).
+You can add a new job to this list, and so that newer jobs appear at the top, we ask
+that you **at the new entry to the top of the list.**
```

Comment:
```suggestion
that you **add the new entry to the top of the list.**
```

Reply:
Looks like I'm dyslexic! Thanks for the fix!